crates/clarirs_num/src/float.rs

```diff
@@ -440,32 +434,34 @@ impl Add for Float {
         let exponent_diff = larger.exponent.len() - smaller.exponent.len();
         let aligned_smaller_mantissa = smaller.mantissa.clone() >> exponent_diff;
 
+        let aligned_smaller_mantissa_val = aligned_smaller_mantissa?;
```

Comment:
Small nitpick but the ? could go at the end of the `aligned_smaller_mantissa` statement